src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,102 @@
 /* eslint-disable max-len */
+import classNames from 'classnames';
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { FilterType } from '../../types/FilterType';
 
-export const TodoList: React.FC = () => {
-  return (
-    <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
+type Props = {
+  isLoading: boolean,
+};
+
+export const TodoList: React.FC<Props> = ({
+  isLoading: loading,
+}) => {
+  const todos = useAppSelector(state => state.todos);
+  const filter = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+
+  const getVisibleTodos = () => {
+    let statusFiltered;
+
+    switch (filter.status) {
+      case FilterType.active:
+        statusFiltered = todos.filter(todo => !todo.completed);
+        break;
+
+      case FilterType.completed:
+        statusFiltered = todos.filter(todo => todo.completed);
+        break;
+
+      default:
+        statusFiltered = todos;
+    }
+
+    return statusFiltered.filter(todo => todo.title.toLocaleLowerCase().includes(filter.query.toLocaleLowerCase().trim()));
```

Comment:
too long line
Split this with variables
Also, you can move this function to a separate file or wrap it with useCallback